pkg/processor/trigger/v3iostream/trigger.go

```diff
@@ -222,6 +225,21 @@ func (vs *v3iostream) ConsumeClaim(session streamconsumergroup.Session, claim st
 	return submitError
 }
 
+func (vs *v3iostream) Abort(session streamconsumergroup.Session) error {
```

Comment:
why both Abort and abort? why not `abort()` only? meaning, why exposing the trigger's api functionality at all. can remain private

Reply:
The public `Abort` is a part of the v3io `Handler` interface.
If we only had a private `abort`, the trigger won't implement the interface.

We can skip the private abort and let `Abort()` call the abstract's `Restart` method.

Reply:
stash my words <<